boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059.rst

```diff
@@ -135,6 +134,48 @@ Here is an example for the :ref:`blinky-sample` application.
 
 Observe the LED on the board blinking.
 
+Flashing with a device firmware update (DFU)
+============================================
+
+The board is factory-programmed with Nordic's bootloader from Nordic's nRF5 SDK.
+This section covers the steps required to migrate to MCUboot and use its serial
+recovery mode over USB CDC ACM to program new firmware.
+
+Obtain a Nordic-signed MCUboot firmware image, then perform a firmware update
+over USB using Nordic's nrfutil command line utility as follows::
+
+	nrfutil dfu usb_serial -pkg mcuboot-img.zip -p /dev/ttyACM0
```

Comment:
Is this normal that this doesn't work at first time. Every time i try to run i have to call the command twice.
```
anpu@ubuntu:~/codedev/mcuboot$ nrfutil dfu usb_serial -pkg mcuboot_nrf52840_pca10059.zip -p /dev/ttyACM0
  [------------------------------------]    0%
Traceback (most recent call last):
  File "/home/anpu/.local/bin/nrfutil", line 11, in <module>
    sys.exit(cli())
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/anpu/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/__main__.py", line 714, in usb_serial
    do_serial(package, port, connect_delay, flow_control, packet_receipt_notification, baud_rate, False)
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/__main__.py", line 680, in do_serial
    dfu.dfu_send_images()
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/dfu/dfu.py", line 125, in dfu_send_images
    self._dfu_send_image(self.manifest.bootloader)
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/dfu/dfu.py", line 90, in _dfu_send_image
    self.dfu_transport.open()
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/dfu/dfu_transport_serial.py", line 213, in open
    self.__set_prn()
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/dfu/dfu_transport_serial.py", line 309, in __set_prn
    self.__get_response(DfuTransportSerial.OP_CODE['SetPRN'])
  File "/home/anpu/.local/lib/python2.7/site-packages/nordicsemi/dfu/dfu_transport_serial.py", line 441, in __get_response
    + 'Expected: 0x{:02X} Received: 0x{:02X}'.format(operation, resp[1]))
pc_ble_driver_py.exceptions.NordicSemiException: Unexpected Executed OP_CODE.
Expected: 0x02 Received: 0x41
anpu@ubuntu:~/codedev/mcuboot$ nrfutil dfu usb_serial -pkg mcuboot_nrf52840_pca10059.zip -p /dev/ttyACM0
  [####################################]  100%             
Device programmed.
```

Reply:
It is not, what version of nrfutil do you have?

Reply:
3.5.1

Reply:
Try to update to the latest version, 4.0.0.